prisma-fmt-wasm/src/lib.rs

```diff
@@ -11,16 +11,22 @@ pub fn get_config(params: String) -> Result<String, JsError> {
     prisma_fmt::get_config(params).map_err(|e| JsError::new(&e))
 }
 
+/// Docs: https://prisma.github.io/prisma-engines/doc/prisma_fmt/fn.get_dmmf.html
```

Comment:
No need to change that now, but the link could be relative, referring to the function by name (https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html)

Reply:
Thanks for the link, I'd keep it as it is atm so that's consistent with the link provided above `get_config()` in the same file